XLS-0056d-batch/README.md

```diff
@@ -0,0 +1,626 @@
+    Amount: "1000",
+    BatchTxn: {
+      OuterAccount: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+      Sequence: 3,
+      BatchIndex: 1
+    },
+    Sequence: 0,
+    Fee: "0",
+    SigningPubKey: "",
+    TxnSignature: ""
+  }
+]
+```
+</details>
+
+### 6.2. Multiple Accounts
+
+In this example, two users are atomically swapping their tokens, XRP for GKO.
+
+#### 6.2.1. Sample Transaction
+
+<details open>
+<summary>
+
+The inner transactions are still not signed, but the `BatchSigners` field is needed on the outer transaction, since there are two accounts' inner transactions in this `Batch` transaction.
+</summary>
+
+```typescript
+{
+  TransactionType: "Batch",
+  Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+  Flags: "1",
+  TxnIDs: [
+    "A2986564A970E2B206DC8CA22F54BB8D73585527864A4484A5B0C577B6F13C95",
+    "0C4316F7E7D909E11BB7DBE0EB897788835519E9950AE8E32F5182468361FE7E"
+  ],
+  RawTransactions: [
+    {
+      RawTransaction: {
+        TransactionType: "Payment",
+        Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+        Destination: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+        Amount: "6000000",
+        BatchTxn: {
+          OuterAccount: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+          Sequence: 4,
+          BatchIndex: 0
+        },
+        Sequence: 0,
+        Fee: "0",
+        SigningPubKey: "",
+        TxnSignature: ""
+      }
+    },
+    {
+      RawTransaction: {
+        TransactionType: "Payment",
+        Account: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+        Destination: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+        Amount: {
+          currency: "GKO",
+          issuer: "ruazs5h1qEsqpke88pcqnaseXdm6od2xc",
+          value: "2"
+        },
+        BatchTxn: {
+          OuterAccount: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+          Sequence: 20,
+          BatchIndex: 0
+        },
+        Sequence: 0,
+        Fee: "0",
+        SigningPubKey: "",
+        TxnSignature: ""
+      }
+    }
+  ],
+  BatchSigners: [
+    {
+      BatchSigner: {
+        Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+        SigningPubKey: "03072BBE5F93D4906FC31A690A2C269F2B9A56D60DA9C2C6C0D88FB51B644C6F94",
+        Signature: "304502210083DF12FA60E2E743643889195DC42C10F62F0DE0A362330C32BBEC4D3881EECD022010579A01E052C4E587E70E5601D2F3846984DB9B16B9EBA05BAD7B51F912B899"
+      }
+    },
+    {
+      BatchSigner: {
+        Account: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+        SigningPubKey: "03C6AE25CD44323D52D28D7DE95598E6ABF953EECC9ABF767F13C21D421C034FAB",
+        Signature: "304502210083DF12FA60E2E743643889195DC42C10F62F0DE0A362330C32BBEC4D3881EECD022010579A01E052C4E587E70E5601D2F3846984DB9B16B9EBA05BAD7B51F912B899"
+      }
+    },
+  ],
+  Sequence: 4,
+  Fee: "60",
+  SigningPubKey: "03072BBE5F93D4906FC31A690A2C269F2B9A56D60DA9C2C6C0D88FB51B644C6F94",
+  TxnSignature: "30440220702ABC11419AD4940969CC32EB4D1BFDBFCA651F064F30D6E1646D74FBFC493902204E5B451B447B0F69904127F04FE71634BD825A8970B9467871DA89EEC4B021F8"
```

Comment:
If it contains Inner Transactions from multiple accounts, shouldn't the two fields `SigningPubKey` and `TxnSignature` be empty string(`""`)?

Reply:
No, the transaction isn't multisigned. The account sender still needs to sign the other fields, like `Fee`.